src/test/java/org/jenkinsci/plugins/googleplayandroidpublisher/ApkPublisherTest.java

```diff
@@ -111,6 +111,7 @@ public void configRoundtripWorks() throws Exception {
         publisher.setGoogleCredentialsId("credential-b");
         publisher.setFilesPattern("**/builds/*.apk, *.aab");
         publisher.setDeobfuscationFilesPattern("**/proguard/*.txt");
+        publisher.setNativeDebugSymbolFilesPattern("**/native/*.zip");
```

Comment:
Would be great to add real tests for these scenarios - especially around the file-count logic in ApkPublisher